core/domain/platform_parameter_registry.py

```diff
@@ -275,9 +276,9 @@ def load_platform_parameter_from_memcache(cls, name):
             PlatformParameter|None. The loaded instance, None if it's not found
             in cache.
         """
-        memcache_key = param_domain.PlatformParameter.get_memcache_key(name)
-        cached_parameter = memcache_services.get_multi([memcache_key]).get(
-            memcache_key)
+        cached_parameter = caching_services.get_multi(
+            caching_services.CACHE_NAMESPACE_PLATFORM, None, [name]
```

Comment:
Prefer PLATFORM_PARAMETER since platform by itself seems a bit too generic for this namespace.

Reply:
Done.